src/Core/src/Handlers/TimePicker/TimePickerHandler.MacCatalyst.cs

```diff
@@ -6,6 +6,8 @@ namespace Microsoft.Maui.Handlers
 {
 	public partial class TimePickerHandler : ViewHandler<ITimePicker, UIDatePicker>
 	{
+		readonly UIDatePickerProxy _proxy = new();
```

Comment:
I found MacCatalyst has a *different* handler than iOS, so I found I had to make fixes in here as well.